.github/ISSUE_TEMPLATE/maintainerRequest.yml

```diff
@@ -0,0 +1,83 @@
+name: Maintainer Request
+description: Request membership to help maintain this repository.
+title: Maintainer Request
+labels: ['maintainer-request']
+projects: ['repo-management']
+assignees:
+  - kilasuit
+body:
+- type: markdown
+  attributes:
+    value: |
+      ## Thank you for your interest in joining as a maintainer of this project. 
+
+      ### Please complete the following public form to request we review your application to join as a maintainer. 
+- type: dropdown  
+  id: time
+  validations:
+    required: true
+  attributes:
+    label: Can you provide at least 1 hour per week to help maintain this repository?
+    description: >-
+      Please select Yes or No.     
+    options:
+      - "Yes"
+      - "No"
+- type: dropdown  
+  id: accesslevel
+  validations:
+    required: true
+  attributes:
+    label: Do you require direct write access to the repository?
+    description: >-
+      This may be required for some maintainer roles. Please select Yes or No.
+      If you select No, you will be added as a collaborator and will not have direct write access to the repository.
+      If you select No, you will **still** be required to create a pull request to make changes to the repository.
+      If you select Maybe, will discuss with you if this is needed & if so how/when this may be granted in future.
+    options:
+      - "Yes"
+      - "No"
+      - "Maybe"
```

Comment:
this is just to show that you can be a maintainer but not need access to the repo with write access to the repo

However, In my experience it's a frustrating experience & this is added as a way to gauge with the following questions what kind of maintainer a requester may want to engage with, both to begin with and in future.